### PR TITLE
PWR002: Add Fortran example using BLOCK

### PR DESCRIPTION
**Pull request by alvrogd**
This PR adds Fortran examples for PWR002 using the `BLOCK` construct, thus solving #1.

Thanks to @Beliavsky and @jalvesz for their feedback!

### CODE REVIEW

**Review by parequena — APPROVED** [review]
Even I understand it, perfect

File: Checks/PWR002/README.md [hunk 1]
```diff
@@ -90,10 +90,72 @@ void example() {
 
 #### Fortran
 
->**Note**  
->The Fortran programming language does not allow to declare variables inside
->loops, in the statements of the loop body. In modern Fortran there are some
->circumstances where this is allowed, such as `do concurrent` statements.
+Fortran 2008 introduced the `BLOCK` construct. This feature is rather
+convenient to create sections of code for enhanced readability in larger
+programs. Conveniently, it supports new variable declarations.
```
Comment (inaki-amatria) [inline Checks/PWR002/README.md:95]:
```suggestion
Fortran 2008 introduced the `BLOCK` construct. This feature allows to organize
code within larger programs by grouping sections together. Conveniently, the
`BLOCK` construct supports declaring variables within those sections, further pro
```

What do you think? This avoids repeating "convenient" in the same paragraph.

**Review by inaki-amatria — APPROVED** [review]
LGTM

**Review by jgonzac — APPROVED** [review]

**Review by ruifm — APPROVED** [review]

File: Checks/PWR002/README.md (hunk 1 above)
Comment (alvrogd) [reply Checks/PWR002/README.md]: Done! :)

**Pull request merged**